haddock: src/Haskoin/Block/Common.hs:278:5: error: parse error on input ‘{’

```
    Running Haddock on library for haskoin-core-0.13.5..
    
    src/Haskoin/Block/Common.hs:278:5: error: parse error on input ‘{’
        |
    278 |     { headersList :: [BlockHeaderCount]
        |     ^
```

--- a/src/Haskoin/Block/Common.hs
+++ b/src/Haskoin/Block/Common.hs
@@ -274,8 +274,9 @@ type BlockHeaderCount = (BlockHeader, VarInt)
 
 -- | The 'Headers' type is used to return a list of block headers in
 -- response to a 'GetHeaders' message.
-newtype Headers = Headers -- | list of block headers with transaction count
-    { headersList :: [BlockHeaderCount]
+newtype Headers = Headers
+    { -- | list of block headers with transaction count
+      headersList :: [BlockHeaderCount]
     } deriving (Eq, Show, Generic, NFData)
 
 instance Serialize Headers where

